nexus/src/app/background/sync_switch_configuration.rs

```diff
@@ -587,6 +590,154 @@ impl BackgroundTask for SwitchPortSettingsManager {
                             bgp_announce_prefixes.insert(bgp_config.bgp_announce_set_id, prefixes);
                         }
 
+                        let ttl: Option<u8> = match peer.min_ttl.as_ref() {
+                            Some(x) => {
+                                match x.0.try_into() {
+                                    Ok(ttl) => Some(ttl),
+                                    Err(e) => {
+                                        error!(log,
+                                            "peer ttl out of range {} > 255",
+                                            x;
+                                            "error" => ?e
```

Comment:
Can be wrapped in slog-error-chain.

Reply:
I went through and tried to do all the error chain display wrapping I could in this area of the code, but there may need to be some broader work to capture all the things.

Reply:
thanks!